docs/error-reporting-usage.rst

```diff
@@ -76,6 +76,19 @@ be used by Stackdriver Error Reporting to help group exceptions.
    >>> except Exception:
    >>>     client.report_exception(http_context=http_context, user=user))
 
+An automatic helper to build the HTTP Context from a Flask (Werkzeug) request
+object is provided.
+
+ .. code-block:: python
+
+    >>> @app.errorhandler(Exception)
+    >>> def handle_error(e):
+    >>>     client = error_reporting.Client()
+    >>>     client.report_exception(
+    >>>         http_context=build_flask_context(request))
+    >>>     # rest of error response code here
```

Comment:
This isn't what the interpreter would print out, i.e. lots of `>>>` need to become `...`. I recommend running your snippets for realsies and then copy-pasting. e.g.

```python
>>> def deco(func):
...     func.salt = 'pepper'
...     return func
... 
>>> @deco
... def f(x, y):
...     return x + y
... 
>>> f(1.0, 2.0)
3.0
>>> f.salt
'pepper'
```